Makefile

```diff
@@ -55,11 +55,14 @@ clean:
 	$(MAKE) -C library clean
 	$(MAKE) -C programs clean
 	$(MAKE) -C tests clean
+ifndef WINDOWS
 	find . \( -name \*.gcno -o -name \*.gcda -o -name *.info \) -exec rm {} +
+endif
```

Comment:
1. Is there a way we can port this cleanup step to windows?
2. There seems to be a slash missing in the `*.info`alternative